snuba/subscriptions/scheduler_processing_strategy.py

```diff
@@ -270,25 +270,13 @@ def submit(self, message: Message[Tick]) -> None:
         # Periodically record the legnth of each buffer
         self.__record_tick_buffer_length()
```

Comment:
What about producing a metric with the low watermark (or the high one, either would work) from every buffer as a counter or gauge? 
Then we can see where we get stuck and if there is a partition that is lagging particularly behind. Just as Kafka record a metric containing the broker offset.